### DISCUSSION TITLE
Adding to the favorites in repo

### DISCUSSION THREAD

**Discussion by emrecoban** [community/community#3592] [Mobile] [answered]
I have to be back main screen for adding any repo to my favorites. I think an add button should be in repo screens.

**Comment by PranamBhat** [comment]
Can you elaborate please

**Reply by emrecoban** [reply]
Of course, for example, when I am in a repo in GitHub mobile app, I cannot add the repo to my favorites in the repo screen. I wish there was a button to add to the favorites.

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback. I've created an issue in our internal repo to keep track of similar requests and evaluate priority from time to time. Please keep the feedback coming!

**Reply by emrecoban** [reply]
Thank you so much. I'm looking forward to the developments.

**Comment by vae520283995** [comment]
@emrecoban Hi, Can the github web version add favorites?

**Reply by emrecoban** [reply]
As far as I know there is no option like that. You can give star to any repo and list them.

**Reply by DSeeLP** [reply]
I have 2 repositories in my favorites view. But i have no clue how i added them

**Comment by town3r** [comment]
How do you add favorites?

**Reply by emrecoban** [reply]
I don't use GitHub mobile for a long time. As far as I remember you can add any repo to your favorites in home screen.

**Reply by NAWABDERA1** [reply]
Thank you very much

**Comment by NAWABDERA1** [comment]
INSHA ALLAH I will be coming back
